operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ControllerConfigurationOverrider.java

```diff
@@ -158,6 +161,11 @@ public ControllerConfigurationOverrider<R> withGenericFilter(GenericFilter<R> ge
     return this;
   }
 
+  public ControllerConfigurationOverrider<R> withItemStore(UnaryOperator<R> cachePruneFunction) {
```

Comment:
This should be called `withCachePruneFunction`

Reply:
fixed